.github/workflows/ci.yml

```diff
@@ -22,59 +26,32 @@ jobs:
         bundle exec rake partial_benchmark
         bundle exec rake translatable_benchmark
   test:
+    name: test (${{ matrix.rails_version }}, ${{ matrix.ruby_version }}, ${{ matrix.mode }})
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        include:
-          - rails_version: "6.1.4.4"
-            ruby_version: "3.0"
-            mode: "capture_patch_enabled"
-          - rails_version: "7.0.2.3"
-            ruby_version: "3.0"
-            mode: "capture_patch_enabled"
-          - rails_version: "main"
-            ruby_version: "3.1"
-            mode: "capture_patch_enabled"
-          - rails_version: "7.1.1"
-            ruby_version: "3.2"
-            mode: "capture_patch_enabled"
-          - rails_version: "main"
-            ruby_version: "3.2"
-            mode: "capture_patch_enabled"
-          - rails_version: "6.1.4.4"
-            ruby_version: "3.0"
-            mode: "capture_patch_disabled"
-          - rails_version: "7.0.2.3"
-            ruby_version: "3.0"
-            mode: "capture_patch_disabled"
-          - rails_version: "main"
-            ruby_version: "3.1"
-            mode: "capture_patch_disabled"
-          - rails_version: "7.1.1"
-            ruby_version: "3.2"
-            mode: "capture_patch_disabled"
-          - rails_version: "main"
-            ruby_version: "3.2"
-            mode: "capture_patch_disabled"
+        rails_version: ["6.1", "7.0", "7.1", "main"]
+        ruby_version: ["3.0", "3.1", "3.2"]
```

Comment:
Is there a reason we were previously selective about which versions we'd run against? Maybe it was a holdover from when we needed specific version combos. That would still be possible [this way](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations).

Reply:
Yeah, certain versions of Rails require certain versions of Ruby.

Reply:
I was wondering if we'd still need to do this, but looking at [endoflife.date](https://endoflife.date/rails), the lists of versions we have here look good.